Monika After Story/game/script-topics.rpy

```diff
@@ -1567,16 +1570,54 @@ label monika_yuri:
         m 2dsc "Just the thought of it makes me shiver."
         m 2eka "But come on...everyone's killed people in games before."
         m "Does that make you a psychopath? Of course not."
-    m 2euc "But if you do happen to be into the yandere type..."
+    m 2euc "But if you do happen to be into the yandere type... {nw}"
     show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
     m 5eua "I can try acting a little more creepy for you. Ehehe~"
-    m "Then again..."
-    show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
-    m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
-    m 2etc "Is this a yandere girl's dream?"
-    if not persistent._mas_pm_cares_about_dokis:
-        m 1eua "I'd ask Yuri if I could."
-    return
+    $ _history_list.pop()
+    menu:
+        m "I can try acting a little more creepy for you. Ehehe~{fast}"
+        "Actually... I'm into yanderes":
+            $ persistent._mas_pm_likes_yanderes = 1
+            m 1sub "Now that I'm hearing that, I'm quite surprised you prefer me over Yuri."
+            if not persistent._mas_pm_cares_about_dokis:
+                m 1cua "Was it that because you realized, that was me, who manipulated everyone and everything to be with you... FOREVER."
```

Comment:
This sentence is a bit odd grammatically. It starts as a question, but by the time it reaches the end, it's a statement. I shortened it a little bit to try to make it flow better.
"Well, it {i}was{/i} me who manipulated everyone and everything to be with you... FOREVER."

Reply:
For lunulae's sentence, it'd be "Well, it {i}was{/i} I who..." not "Well, it {i}was{/i} me who..."